### PR TITLE
Update webpack proxy and fix namespace issue for KCP

### PR DESCRIPTION
**Pull request by rohitkrai03**
## Fixes 

- https://issues.redhat.com/browse/HAC-2197

## Description

This PR 
- Adds a custom proxy for KCP.
- Adds a new npm script to run for KCP.
- Fixes the namespace selection utility when running on KCP.

## Type of change

- [x] Feature

## Screen shots / Gifs for design review 

No UI change.

## How to test or reproduce?

- Setup KCP locally.
- Follow the KCP onboarding guide or the App Studio quick start guide for KCP.

## Browser conformance: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: config/dev.webpack.config.js [hunk 1]
```diff
@@ -17,6 +17,49 @@ const calculateRemoteConfig = (remoteConfig) => {
...
+    target: process.env.KCP_SERVER,
+    secure: false,
+    changeOrigin: true,
+    autoRewrite: true,
+    ws: true,
+    pathRewrite: { '^/api/k8s': '' },
+  },
+  {
+    context: (path) => path.includes('/wss/k8s'),
+    target: process.env.KCP_SERVER.replace('https://', 'wss://'),
```
Comment (christianvogt) [inline config/dev.webpack.config.js:32]: This line throws an error when running `npm run start:prod:beta` because `process.env.KCP_SERVER` is `undefined`.

**Comment by christianvogt** [comment]
KCP start script works!

**Comment by karthikjeeyar** [comment]
Tested kcp and non-kcp environments, both works as expected.

**Pull request merged**